scripts/put-azure-credentials.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Why do we need a separate script for it?

Reply:
I suppose that's because this logic is not to be used everywhere; Can probably be fixed to use function and a condition

Reply:
Also, this script is used for local setting these values.